src/test/rates.sol

```diff
@@ -5,46 +5,46 @@ contract Rates {
     mapping (uint256 => uint256) public rates;
 
     constructor() public {
-        rates[0] = 1000000000000000000000000000;
-        rates[25] = 1000000000079175551708715274;
-        rates[50] = 1000000000158153903837946257;
-        rates[75] = 1000000000236936036262880196;
-        rates[100] = 1000000000315522921573372069;
-        rates[125] = 1000000000393915525145987602;
-        rates[150] = 1000000000472114805215157978;
-        rates[175] = 1000000000550121712943459312;
-        rates[200] = 1000000000627937192491029810;
-        rates[225] = 1000000000705562181084137268;
-        rates[250] = 1000000000782997609082909351;
-        rates[275] = 1000000000860244400048238898;
-        rates[300] = 1000000000937303470807876289;
-        rates[325] = 1000000001014175731521720677;
-        rates[350] = 1000000001090862085746321732;
-        rates[375] = 1000000001167363430498603315;
-        rates[400] = 1000000001243680656318820312;
-        rates[425] = 1000000001319814647332759691;
-        rates[450] = 1000000001395766281313196627;
-        rates[475] = 1000000001471536429740616381;
-        rates[500] = 1000000001547125957863212448;
-        rates[525] = 1000000001622535724756171269;
-        rates[550] = 1000000001697766583380253701;
-        rates[575] = 1000000001772819380639683201;
-        rates[600] = 1000000001847694957439350562;
-        rates[625] = 1000000001922394148741344865;
-        rates[650] = 1000000001996917783620820123;
-        rates[675] = 1000000002071266685321207000;
-        rates[700] = 1000000002145441671308778766;
-        rates[725] = 1000000002219443553326580536;
-        rates[750] = 1000000002293273137447730714;
-        rates[775] = 1000000002366931224128103346;
-        rates[800] = 1000000002440418608258400030;
-        rates[825] = 1000000002513736079215619839;
-        rates[850] = 1000000002586884420913935572;
-        rates[875] = 1000000002659864411854984565;
-        rates[900] = 1000000002732676825177582095;
-        rates[925] = 1000000002805322428706865331;
-        rates[950] = 1000000002877801985002875644;
-        rates[975] = 1000000002950116251408586949;
+        rates[0]    = 1000000000000000000000000000;
```

Comment:
If we're mucking with the alignment here anyway, I think the most aesthetic choice is to line up the end brackets and right-align the numbers so their digits places lines up (i.e. all coefficients of 10^N in the same column for N=1,2,3,...):

```
rates[   0] = XXX;
...
rates[  10] = YYY;
...
rates[ 100] = ZZZ;
...
rates[1000] = QQQ;
```

Reply:
done